packages/contracts/token_contract/tests/utils/mod.rs

```diff
@@ -0,0 +1,188 @@
+use fuels::{prelude::*, types::*};
+
+abigen!(Contract(
+    name = "TokenContract",
+    abi = "packages/contracts/token_contract/out/debug/token_contract-abi.json"
+));
+
+pub struct SetupParams {
+    pub num_wallets: Option<u64>,
+    pub coins_per_wallet: Option<u64>,
+    pub coin_amount: Option<u64>,
+    pub mint_price: Option<u64>,
+    pub mint_limit: Option<u64>,
+    pub total_supply: Option<u64>,
+}
+
+pub struct ConstructorParams {
+    pub mint_price: u64,
+    pub mint_limit: u64,
+    pub total_supply: u64,
+}
+
+pub struct Metadata {
+    pub contract_id: Bech32ContractId,
+    pub asset_id: ContractId,
+    pub token_contract: TokenContract,
+    pub deployer: WalletUnlocked,
+    pub user: WalletUnlocked,
+    pub params: ConstructorParams,
+    pub provider: Provider,
+}
+
+pub mod paths {
+    pub const ASSET_CONTRACT_BINARY_PATH: &str = "./out/debug/token_contract.bin";
+    pub const ASSET_CONTRACT_STORAGE_PATH: &str = "./out/debug/token_contract-storage_slots.json";
+}
```

Comment:
If the paths are not used outside of this file then they do not need to be public, similarly with anything else in the file. I would move it out of the `paths` mod and have them as regular constants just below the imports at the top of the file